examples/14.getWithdrawals.js

```diff
@@ -22,22 +21,19 @@ const dvfConfig = {
   api: 'https://api.deversifi.dev'
 }
 
-
 ;(async () => {
   const dvf = await DVF(web3, dvfConfig)
 
   const getWithdrawalsResponse = await dvf.getWithdrawals()
 
   console.log('getWithdrawals response ->', getWithdrawalsResponse)
-
 })()
-// Stop provider to allow process to exit.
-.then(() => {
-  console.log('Stopping provider...')
-  provider.engine.stop()
-})
-.catch(error => {
-  console.error(error)
-  process.exit(1)
-})
-
+  // Stop provider to allow process to exit.
+  .then(() => {
+    console.log('Stopping provider...')
+    provider.engine.stop()
+  })
+  .catch(error => {
+    console.error(error)
+    process.exit(1)
+  })
```

Comment:
I see only style changes in this file. Tbh I find them unnecessary. If they where forced by the linter can we adjust the longer setting to accept the file as it was? If not, I think all examples should be adjusted to be consistent.

Reply:
Yes this was done by linter. No code was modified. Do you have the linter config we want to use?

Reply:
Thanks for the notes on how to update examples. will do do the updates accordingly.

Reply:
I don't have a linter config, but it seems like there is already a linter configured in the repo (with lint script using [`standard`](https://standardjs.com/) which uses [this config](https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json)), and it does not seem to complain about the style in the `examples/14.getWithdrawals.js` as it is currently on master (tested by running `yarn lint examples/14.getWithdrawals.js`). Maybe your editor is configured to lint with different settings? Anyway, if you use the script for generating examples is should make all of them consistent.